### DISCUSSION TITLE
Weekly Meeting 15 Nov. 2023

### DISCUSSION THREAD

**Discussion by kikuchiharuma** [project-theory-probe/ptp-voyage#7] [General]
Meet: Project Theory Probe Weekly Meeting
Wednesday, November 15 · 14:00 – 15:00
Time zone: Asia/Tokyo
Google Meet joining info
Video call link: https://meet.google.com/qyd-seed-aew
Or dial: ‪(JP) +81 3-4545-0450‬ PIN: ‪100 286 257 6031‬#
More phone numbers: https://tel.meet/qyd-seed-aew?pin=1002862576031

**Comment by kikuchiharuma** [comment]
ハッカー文化の伝道師 2 エリック・レイモンド
書いてる
https://github.com/project-theory-probe/ptp-voyage/discussions/3#discussioncomment-7571467

**Comment by shotaro8yagi** [comment]
アジャイルの清流（川上）にLinuxの成功と、それを伝えようとしたエリックレイモンドが居たという話はどこかでまた話したい

**Comment by kikuchiharuma** [comment]
Obsidian Publish の id 変更

**Reply by kikuchiharuma** [reply]
done
https://github.com/project-theory-probe/ptp-voyage/commit/0625ae918689d956164d4ad2478737c6016b256d

**Comment by kikuchiharuma** [comment]
@shotaro8yagi さんが書いてくれた [規範と自己言及（自己指示）について](https://publish.obsidian.md/ptp/shotaroyagi/%E8%A6%8F%E7%AF%84%E3%81%A8%E8%87%AA%E5%B7%B1%E8%A8%80%E5%8F%8A%EF%BC%88%E8%87%AA%E5%B7%B1%E6%8C%87%E7%A4%BA%EF%BC%89%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6)

あまり話す時間がなかったのでいずれしっかり話したい
ダブル・ダイヤモンド、4d Process Model 、フェーズ・ジャーニー、W型問題解決などが共通して想定している複数のフェーズ、その境界に何があるのか、なぜ複数のフェーズがあるのか、を深掘りしている気がする、が第一印象

**Reply by kikuchiharuma** [reply]
そしてこんな感じで参照できるので Obsidian Publish はやっぱり最高ですね…
はやくリポジトリを公開したい

**Comment by shotaro8yagi** [comment]
ネタメモ：
- H2ロケット←→イーロンマスク
	- https://www.youtube.com/watch?v=4UgHtFKALZU
- 探査機ルーシー
	- https://www.space.com/nasa-lucy-asteroid-dinky-3-space-rocks-contact-binary
- ボイジャー号
	- もっと早く終わるはずだった
	- ヘリオポーズを越えたのは副産物
	- https://www.sciencealert.com/nasa-probe-discovers-a-hidden-surprise-during-asteroid-flyby
- 19世紀ライト兄弟
	- https://www.youtube.com/watch?v=oVUjh0O7YEc
	- 未知のものにゴール設定する虚しさ
- インターネットの歴史
	- [https://en.wikipedia.org/wiki/Gopher_(protocol](https://en.wikipedia.org/wiki/Gopher_%28protocol))
- 深海
	- https://www.youtube.com/watch?v=8QH5hBOoz08
- 山の上
	- https://www.youtube.com/watch?v=8QH5hBOoz08
